pypowsybl/network.py

```diff
@@ -361,6 +365,57 @@ def update_phase_tap_changers(self, df: pd.DataFrame):
         """
         return self.update_elements(_pypowsybl.ElementType.PHASE_TAP_CHANGER, df)
 
+    def get_working_variant_id(self):
```

Comment:
Can you just add some content to the docstrings ? Just a quick description of the method+argument+returned.